utils/gene_symbol_lookup.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import pickle
+
+
+def get_gene_symbol(ensembl_id):
+    url = (
+        "https://rest.ensembl.org/lookup/id/"
+        + ensembl_id
+        + "?content-type=application/json"
+    )
+    data = requests.get(url).json()
+    gene_symbol = data.get("display_name")
+    return gene_symbol
+
+
+def get_ensembl_id(gene_symbol):
+    updated_ensembl_id = None
+    url = (
+        "https://rest.ensembl.org/xrefs/symbol/homo_sapiens/"
+        + gene_symbol
+        + "?content-type=application/json"
+    )
+    data = requests.get(url).json()
+    if data:
+        for item in data:
+            if item["id"].startswith("ENSG"):
+                updated_ensembl_id = item["id"]
+                break
+    return updated_ensembl_id
+
+
+def main():
```

Comment:
It might be more clean if the separate branches of logic here were put in their own functions. Also easier to test smaller pieces of functionality.